src/ast/mod.rs

```diff
@@ -1012,6 +1012,15 @@ pub enum Statement {
         table_name: ObjectName,
         filter: Option<ShowStatementFilter>,
     },
+    /// SHOW TABLES
```

Comment:
I reviewed and this appears to match https://dev.mysql.com/doc/refman/8.0/en/show-tables.html